apps/src/lab2/utils/multiFileSourceEditUtils.ts

```diff
@@ -46,6 +48,10 @@ export const createNewFileHelper = (
     file.url = url;
   }
 
+  if (isStartMode) {
```

Comment:
We need files added via start mode that have starter asset URLs to be tagged as starter assets so that we know not to try and delete them from S3 if a student wants to delete them from their project (ie, it would delete from their main.json, but not attempt the S3 delete). This is an edge case (ie, I imagine levelbuilders will mostly make these file types undeletable), but it seems like a good idea anyway to start giving starter files the correct type (currently the type is typically undefined).